site/content/docs/user/networking-scenarios.md

```diff
@@ -0,0 +1,229 @@
+
+## Kubernetes Networking
+
+<img src="/docs/user/images/kind-kubernetes-network-kindnet.png"/>
+
+[The Kubernetes network model](https://kubernetes.io/docs/concepts/cluster-administration/networking/#the-kubernetes-network-model) implies end to end connectivity without NAT between Pods.
+
+By default, KIND uses its own CNI plugin, **Kindnet**, that install the corresponding routes and iptables rules on the cluster nodes.
+
+## Services
+
+[Kubernetes Services](https://kubernetes.io/docs/concepts/services-networking/service/) are an abstract way to expose an application running on a set of Pods as a network service.
+
+The are different types of Services:
+
+* Cluster IP
+* NodePort
+* LoadBalancer
+* Headless
+* ExternalName
+
+In Linux hosts, you can access directly the Cluster IP address of the services just adding one route to the configured **serviceSubnet** parameters via any of the nodes that belong to the cluster, so there is no need to use NodePort or LoadBalancer services.
+
+## Multiple clusters
+
+As we explained before, all KIND clusters are sharing the same Docker network, that means that all the cluster nodes have direct connectivity.
+
+If we want to spawn multiple clusters and provide Pod to Pod connectivity between different clusters, first we have to configure the cluster networking parameters to avoid address overlapping.
+
+### Example: Kubernetes multi-region
+
+Let's take an example emulating 2 clusters: A and B.
+
+For cluster A we are going to use the following network parameters:
+
+{{< codeFromInline lang="bash" >}}
+cat <<EOF | kind create cluster --name clusterA --config=-
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+networking:
+  podSubnet: "10.110.0.0/16"
+  serviceSubnet: "10.115.0.0/16"
+nodes:
+- role: control-plane
+- role: worker
+EOF
+{{< /codeFromInline >}}
+
+And Cluster B:
+
+{{< codeFromInline lang="bash" >}}
+cat <<EOF | kind create cluster --name clusterB --config=-
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+networking:
+  podSubnet: "10.220.0.0/16"
+  serviceSubnet: "10.225.0.0/16"
+nodes:
+- role: control-plane
+- role: worker
+EOF
+{{< /codeFromInline >}}
+
+All the nodes in each cluster will have routes to the podsSubnets assigned to the nodes of the same cluster.
+If we want to provide Pod to Pod connectivity between different clusters we just have to do the same in each node.
+
+We can obtain the routes using kubectl:
+
+{{< codeFromInline lang="bash" >}}
+$ kubectl --context kind-clusterA get nodes -o=jsonpath='{range .items[*]}{"ip route add "}{.spec.podCIDR}{" via "}{.status.addresses[?(@.type=="InternalIP")].address}{"\n"}{end}'
+ip route add 10.110.0.0/24 via 172.17.0.4
+ip route add 10.110.1.0/24 via 172.17.0.3
+ip route add 10.110.2.0/24 via 172.17.0.2
+
+$kubectl --context kind-clusterB get nodes -o=jsonpath='{range .items[*]}{"ip route add "}{.spec.podCIDR}{" via "}{.status.addresses[?(@.type=="InternalIP")].address}{"\n"}{end}'
+ip route add 10.120.0.0/24 via 172.17.0.7
```

Comment:
Is this supposed to be 220? Also why are there three results here when each cluster has two nodes?

Reply:
heh, good catch on both things, is 220 and the config should have 3 nodes